modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/FDS/APIApplicationsSchemasTable.tsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {FrontendDataSet} from '@liferay/frontend-data-set-web';
+import React from 'react';
+
+import {getAPIApplicationsSchemasFDSProps} from './fdsUtils/schemasFDSProps.';
+
+interface APIApplicationsTableProps {
+	apiURLPaths: APIURLPaths;
+	portletId: string;
+	readOnly: boolean;
+}
+
+export default function APIApplicationsSchemasTable({
+	apiURLPaths,
+	portletId,
+}: APIApplicationsTableProps) {
+	const createAPIApplication = {
```

Comment:
Should not be `createAPISchema`?

Reply:
Or `createAPIApplicationSchema`?

Reply:
I went for `createAPIApplicationSchema` and also edited the equivalent variable in [endpoints](https://github.com/liferay-headless/liferay-portal/pull/1359/commits/cd1e804b914cbfd8e265b161aaa8ac738290e925#diff-5f33c590d98d915c75e3d041e0d3b324c9678e508a227424f656c613c727d5a8R32).